xdc-plugin/tests/counter_golden.json

```diff
@@ -0,0 +1 @@
+{"OBUF_7": {"IN_TERM": "UNTUNED_SPLIT_40", "SLEW": "FAST", "IOSTANDARD": "SSTL135"}, "OBUF_6": {"IOSTANDARD": "LVCMOS33", "DRIVE": "I12", "SLEW": "SLOW"}, "bottom_inst.OBUF_9": {"IOSTANDARD": "SSTL135", "SLEW": "FAST"}, "OBUF_OUT": {"IN_TERM": "UNTUNED_SPLIT_40", "SLEW": "FAST", "IOSTANDARD": "SSTL135"}, "bottom_inst.OBUF_11": {"IOSTANDARD": "SSTL135", "SLEW": "FAST"}, "bottom_inst.OBUF_10": {"IOSTANDARD": "SSTL135", "SLEW": "FAST"}, "bottom_intermediate_inst.OBUF_8": {"IOSTANDARD": "SSTL135", "SLEW": "FAST"}}
```

Comment:
Please use "indent=2" on the `json.dumps` to enable better readability.

Reply:
Done